core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
I have a feeling our entire `akka` block belongs in `application.conf` (@kshakir you're the expert on these things). Just keep it here since the rest of the akka stuff is here and we can address separately